src/main/resources/db/migration/V20251208.2848__add_mahdollisuus_view.sql

```diff
@@ -0,0 +1,42 @@
+CREATE OR REPLACE VIEW mahdollisuus_view AS
+SELECT
+  m.id AS id,
+  m.tyyppi AS tyyppi,
+  m.ammattiryhma AS ammattiryhma,
+  m.aineisto AS aineisto,
+  m.otsikko  AS otsikko,
+  m.koulutusTyyppi AS koulutus_tyyppi,
+  STRING_AGG(DISTINCT COALESCE(m.maakunta, ''), ', ') AS maakunnat,
```

Comment:
- distinct is unnecessary (the distribution values are guaranteed to be unique). 
- coalesce is unnecessary, string_agg ignores nulls- 
- you could use `array_agg(m.maakunta) filter (where m.maakunta is not null)` and bind it to a String[], avoiding joining and splitting and the assumption that codes no not include commas.